From e33edc4881bd76b8d37e31b1f1a5da0af1b1bcf6 Mon Sep 17 00:00:00 2001
From: Rasmus Thomsen <oss@cogitri.dev>
Date: Tue, 20 Dec 2022 13:28:29 +0100
Subject: [PATCH] Adjust OpenJPEG variables to FindOpenJPEG.cmake

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 95c33040..2895704a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -250,7 +250,7 @@ set(WITH_OPENJPEG FALSE)
 if(ENABLE_LIBOPENJPEG STREQUAL "openjpeg2")
   find_package(OpenJPEG REQUIRED)
   set(WITH_OPENJPEG ${OpenJPEG_FOUND})
-  if(NOT OpenJPEG_FOUND OR OPENJPEG_MAJOR_VERSION VERSION_LESS 2)
+  if(NOT OpenJPEG_FOUND OR OpenJPEG_VERSION_MAJOR VERSION_LESS 2)
     message(STATUS "Could NOT find openjpeg2.")
     message(FATAL_ERROR "Install libopenjpeg2 before trying to build poppler. You \
 can also decide to use the internal unmaintained JPX decoder or none at all.\n\
@@ -502,7 +502,7 @@ if (OpenJPEG_FOUND)
   )
   set(poppler_LIBS ${poppler_LIBS} openjp2)
   # target openjp2 may lack interface include directories
-  include_directories(SYSTEM ${OPENJPEG_INCLUDE_DIRS})
+  include_directories(SYSTEM ${OpenJPEG_INCLUDE_DIR})
 else ()
   set(poppler_SRCS ${poppler_SRCS}
     poppler/JPXStream.cc
-- 
2.37.1 (Apple Git-137.1)

