diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3bd869a9..bd63bbb3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -631,6 +631,11 @@ endif()
 if(USE_CMS)
   target_include_directories(poppler SYSTEM PRIVATE ${LCMS2_INCLUDE_DIR})
 endif()
+if(WITH_FONTCONFIGURATION_FONTCONFIG)
+  target_include_directories(poppler SYSTEM PRIVATE ${Fontconfig_INCLUDE_DIR})
+  target_link_libraries(poppler PRIVATE Fontconfig::Fontconfig)
+endif()
+
 generate_export_header(poppler BASE_NAME poppler-private EXPORT_FILE_NAME "${CMAKE_CURRENT_BINARY_DIR}/poppler_private_export.h")
 set_target_properties(poppler PROPERTIES
                               VERSION ${POPPLER_SOVERSION_NUMBER}.0.0
