From e33edc4881bd76b8d37e31b1f1a5da0af1b1bcf6 Mon Sep 17 00:00:00 2001
From: Rasmus Thomsen <oss@cogitri.dev>
Date: Tue, 20 Dec 2022 13:28:29 +0100
Subject: [PATCH] Adjust OpenJPEG variables to FindOpenJPEG.cmake

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3de278ef..83db90f4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -288,7 +288,7 @@ set(WITH_OPENJPEG FALSE)
 if(ENABLE_LIBOPENJPEG STREQUAL "openjpeg2")
   find_package(OpenJPEG REQUIRED)
   set(WITH_OPENJPEG ${OpenJPEG_FOUND})
-  if(NOT OpenJPEG_FOUND OR OPENJPEG_MAJOR_VERSION VERSION_LESS 2)
+  if(NOT OpenJPEG_FOUND OR OpenJPEG_VERSION_MAJOR VERSION_LESS 2)
     message(STATUS "Could NOT find openjpeg2.")
     message(FATAL_ERROR "Install libopenjpeg2 before trying to build poppler. You \
 can also decide to use the internal unmaintained JPX decoder or none at all.\n\
@@ -595,7 +595,7 @@ add_library(poppler ${poppler_SRCS})
 if (OpenJPEG_FOUND)
   # check if we can remove this when we depend on newer openjpeg versions, 2.5 seems fixed
   # target openjp2 may lack interface include directories
-  target_include_directories(poppler SYSTEM PRIVATE ${OPENJPEG_INCLUDE_DIRS})
+  target_include_directories(poppler SYSTEM PRIVATE ${OpenJPEG_INCLUDE_DIR})
 endif()
 if(USE_CMS)
   target_include_directories(poppler SYSTEM PRIVATE ${LCMS2_INCLUDE_DIR})
-- 
2.37.1 (Apple Git-137.1)

