From 1769a0233a23073de5f42b98c9974ea55f134b3d Mon Sep 17 00:00:00 2001
From: Rasmus Thomsen <oss@cogitri.dev>
Date: Sat, 24 Dec 2022 15:47:22 +0100
Subject: [PATCH] Use CMake's version of FindIconv to fix build on MacOS

---
 CMakeLists.txt     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d1d3653a..dc4fac65 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -227,8 +227,8 @@ endif()

 if(ENABLE_CPP)
   macro_optional_find_package(Iconv)
-  set(ENABLE_CPP ${ICONV_FOUND})
-  set(HAVE_ICONV ${ICONV_FOUND})
+  set(ENABLE_CPP ${Iconv_FOUND})
+  set(HAVE_ICONV ${Iconv_FOUND})
 endif()
 if(ENABLE_ZLIB)
   find_package(ZLIB REQUIRED)
-- 
2.37.1 (Apple Git-137.1)

