diff --git a/CMakeLists.txt b/CMakeLists.txt
index d1d3653a..6625afe4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -137,6 +137,7 @@ set(GTK_REQUIRED "3.22")
 set(GDK_PIXBUF_REQUIRED "2.36")
 
 find_package(Freetype REQUIRED)
+include_directories(SYSTEM ${FREETYPE_INCLUDE_DIR_freetype2} ${FREETYPE_INCLUDE_DIR_ft2build})
 if(WITH_FONTCONFIGURATION_FONTCONFIG)
   find_package(Fontconfig REQUIRED)
 endif()
